src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -232,6 +232,26 @@ export function queryForFacilities(input = '') {
     });
 }
 
+export function getSeparationLocations() {
+  return apiRequest('/disability_compensation_form/separation_locations')
+    .then((
+      { separation_locations }, // eslint-disable-line camelcase
+    ) =>
+      // eslint-disable-next-line camelcase
+      separation_locations.map(separationLocation => ({
+        id: separationLocation.code,
+        label: separationLocation.description,
+      })),
+    )
+    .catch(error => {
+      Sentry.withScope(scope => {
+        scope.setExtra('error', error);
+        Sentry.captureMessage('Error getting separation locations');
+      });
+      return [];
```

Comment:
Does this prevent the user from proceeding? Can we then pull the list from somewhere else, like the vets-json-schema constants? And what happens if the user is allowed to proceed, but has entered an incorrect match?

Reply:
I made a note about this above, but yeah it would prevent them from continuing. I think I need to connect with the back end engineers to see what they think about some default responses and where those should live.

Reply:
But I think I remember Anna saying the response is cached on the back end, so it might be true that even when the EVSS service is down we will still have some response.